packages/api-client/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@tupaia/api-client",
-  "version": "3.0.0",
+  "version": "3.1.0",
```

Comment:
Not exactly sure how to manage this version bumping of api-client. In theory we only need to bump it when we publish a new version, which is really up to us since it's just Tamanu that is using it... I'm bumping it here, but it may be the last time I do so until all the changes for RN-486 are in...